src/UI/V2/SummaryListItem.elm

```diff
@@ -0,0 +1,244 @@
+module UI.V2.SummaryListItem exposing
+    ( SummaryListItem, summaryListItem
+    , withSelected, withBadge, withCheckbox
+    , renderElement
+    )
+
```

Comment:
what do you think about creating this as `UI.ListView.SummaryItem` or something on that line? So that later we don't have to change the modules again when we get rid of the original summary item and also this way it makes sense considering that both components are tied.